model/pdata/generated_metrics.go

```diff
@@ -1192,18 +1192,18 @@ func (es DoubleDataPointSlice) RemoveIf(f func(DoubleDataPoint) bool) {
 // Must use NewDoubleDataPoint function to create new instances.
 // Important: zero-initialized instance is not valid for use.
 type DoubleDataPoint struct {
```

Comment:
What's the future plan? Will this struct be renamed `NumberDataPoint`?

Reply:
I plan on renaming this next and tackling removing IntDataPoint at the same time.